workshop/exercise-1/README.md

```diff
@@ -49,59 +49,59 @@ Learn how to set the context to work with your cluster by using the `kubectl` CL
 
 1.  Set the context for your cluster in your CLI. Every time you log in to the IBM Cloud Kubernetes Service CLI to work with the cluster, you must run these commands to set the path to the cluster's configuration file as a session variable. The Kubernetes CLI uses this variable to find a local configuration file and certificates that are necessary to connect with the cluster in IBM Cloud.
 
-a. List the available clusters.
+    a. List the available clusters.
 
-```shell
-ibmcloud cs clusters
-```
+    ```shell
+    ibmcloud cs clusters
+    ```
 
-b. Download the configuration file and certificates for your cluster using the `cluster-config` command.
+    b. Download the configuration file and certificates for your cluster using the `cluster-config` command.
 
-```shell
-ibmcloud cs cluster-config <your_cluster_name>
-```
+    ```shell
+    ibmcloud cs cluster-config <your_cluster_name>
+    ```
 
-c. Copy and paste the output command from the previous step to set the `KUBECONFIG` environment variable and configure your CLI to run `kubectl` commands against your cluster.
+    c. Copy and paste the output command from the previous step to set the `KUBECONFIG` environment variable and configure your CLI to run `kubectl` commands against your cluster.
 
 2.  Get basic information about your cluster and its worker nodes. This information can help you manage your cluster and troubleshoot issues.
 
-a.  View details of your cluster.
+    a.  View details of your cluster.
 
-```shell
-ibmcloud cs cluster-get <your_cluster_name>
-```
+    ```shell
+    ibmcloud cs cluster-get <your_cluster_name>
+    ```
 
-b.  Verify the worker nodes in the cluster.
+    b.  Verify the worker nodes in the cluster.
 
-```shell
-ibmcloud cs workers <your_cluster_name>
-ibmcloud cs worker-get <worker_ID>
-```
+    ```shell
+    ibmcloud cs workers <your_cluster_name>
+    ibmcloud cs worker-get <worker_ID>
+    ```
 
 3.  Validate access to your cluster.
 
-a.  View nodes in the cluster.
+    a.  View nodes in the cluster.
 
-```shell
-kubectl get node
-```
+    ```shell
+    kubectl get node
+    ```
 
-b.  View services, deployments, and pods.
+    b.  View services, deployments, and pods.
 
-```shell
-kubectl get svc,deploy,po --all-namespaces
-```
+    ```shell
+    kubectl get svc,deploy,po --all-namespaces
+    ```
 
 ## Clone the lab repo
 
-From your command line, run:
+1. From your command line, run:
```

Comment:
For consistency this is the right thing to do... I just wish we'd stop numbering things unless we actually refer to the numbers, which we rarely do. I find the numbering distracting for some reason